app/locales/en/translations.ts

```diff
@@ -1090,7 +1090,7 @@ export default {
             expand: 'Expand',
             see_more: 'See more',
             metadata: 'Metadata',
-
+            component_of: 'This is a component of the registration <a href={{rootUrl}}>{{rootTitle}}</a>',
```

Comment:
Hm, we don't want to reload the whole app to transition to another registration. Maybe change the translation string to `'This is a component of a registration:'` and use `OsfLink` to link.